.travis.yml

```diff
@@ -53,3 +53,7 @@ jobs:
       script:
         - PKG_OS=linux make packages
         - DOCKER_PUSH_LATEST=true make docker-push
+    - name: 'Deploy to staging'
+      stage: release
+      script:
+        - HELM_RELEASE=lookout-gometalint-analyzer HELM_CHART=src-d/lookout-gometalint-analyzer K8S_NAMESPACE=lookout HELM_ARGS="--tiller-namespace=lookout --set image.tag=$TRAVIS_TAG -f .helm_staging.yml" make deploy
```

Comment:
I don't think this is going to work as we need the repo url. `HELM_CHART` should we set to  `lookout-gometalint-analyzer` and we should add `--repo https://src-d.github.io/charts/index.yaml`

Reply:
🤦‍♀️